pkgs/build-support/rust/build-rust-package/default.nix

```diff
@@ -154,7 +155,28 @@ stdenv.mkDerivation ((removeAttrs args [ "depsExtraArgs" "cargoUpdateHook" "carg
 
   strictDeps = true;
 
-  passthru = { inherit cargoDeps; } // (args.passthru or {});
+  passthru = {
+    inherit cargoDeps;
+    sbom = stdenv.mkDerivation {
```

Comment:
could be worth renaming that like `cyclonedx` or `sbom.cyclonedx`

Reply:
Does this sbom include all dependencies (including e.g. C libraries), or just Rust libraries?  If just Rust, maybe calling it `sbom` or `sbom.cyclonedx` would be confusing, since that sounds like something not language-specific?

Reply:
This field encodes for the dependencies that are missing from nix purview. C libraries should be in nix definition.
So yeah, only encodes for the rust libraries.